torch_glow/tests/utils.py

```diff
@@ -1,17 +1,25 @@
+from argparse import Namespace
+
 import torch
 import torch_glow
 
+
 GLOW_NODE_NAME = "glow::CompilationGroup"
+SUBGRAPH_ATTR = "Subgraph"
```

Comment:
delete

Reply:
Still need it now that we want to check that all ops in `expected_fused_ops` are present in the graph.